src/librustc_passes/ast_validation.rs

```diff
@@ -101,7 +106,8 @@ impl<'a> Visitor for AstValidator<'a> {
             ItemKind::Use(ref view_path) => {
                 let path = view_path.node.path();
                 if !path.segments.iter().all(|segment| segment.parameters.is_empty()) {
-                    self.err_handler().span_err(path.span, "type or lifetime parameters \
+                    self.err_handler().span_err(path.span,
+                                                "type or lifetime parameters \
                                                             in import path");
```

Comment:
Its because Rustfmt is putting the string on a newline, but won't touch the indent of the next line because it is inside a string literal. Needs manual adjustment (Rustfmt needs work around string literals).